cupy/manipulation/transpose.py

```diff
@@ -36,6 +36,30 @@ def swapaxes(a, axis1, axis2):
     return a.swapaxes(axis1, axis2)
 
 
+def moveaxis(a, source, destination):
+    """Moves axes of an array to new positions.
+
+    Other axes remain in their original order.
+
+    Args:
+        a (cupy.ndarray): Array whose axes should be reordered.
+        source (int or sequence of int):
```

Comment:
Is int supported?

Reply:
Sorry, I fixed `normalize_axis_tuple` to support `int` and added the test.